src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -1704,14 +1708,16 @@ def join_currents(table: str, bus_name: str, i_name: str) -> pd.DataFrame:
         )
         pp_switches_output = pp_switches_output[["i_ka"]]
         pp_switches_output.set_index(pp_switches_output_index, inplace=True)
-        pp_switches_output["loading_percent"] = np.nan
 
         # For et=b, ie bus to bus switches, links are created. get result from them
         if not links_absent:
             links = self.pgm_output_data[ComponentType.link]
             # For links, i_from = i_to = i_ka / 1e3
             link_ids = self._get_pp_ids("switch", links["id"], "b2b_switches")
             pp_switches_output.loc[link_ids, "i_ka"] = links["i_from"] * 1e-3
+        in_ka = self.pp_input_data["switch"]["in_ka"].values
+        pp_switches_output["loading_percent"] = np.nan
+        pp_switches_output["loading_percent"] = np.divide(pp_switches_output["i_ka"], in_ka, where=in_ka != 0)
```

Comment:
I will double-check tomorrow that this does what it should, or whether it should instead be something along the lines of

```suggestion
        pp_switches_output["loading_percent"] = np.nan
        pp_switches_output["loading_percent"][in_ka != 0] = np.divide(pp_switches_output["i_ka"], in_ka, where=in_ka != 0)
```